src/System.Memory/tests/Span/Reflection.cs

```diff
@@ -0,0 +1,224 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Buffers;
+using System.Buffers.Binary;
+using System.Globalization;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace System.SpanTests
+{
+    public static partial class SpanTests
+    {
+        // Calling Span APIs via Reflection is not supported yet.
+        // These tests check that using reflection results in graceful failures. See https://github.com/dotnet/coreclr/issues/17296
+
+        [Fact]
+        public static void MemoryExtensions_AsSpan()
+        {
+            Type type = typeof(MemoryExtensions);
+
+            MethodInfo method = type.GetMethod("AsSpan", new Type[] { typeof(string) });
```

Comment:
Can we use `nameof(MemoryExtensions.AsSpan)` instead of manually typing the string?  Same question for the rest of these.

Reply:
What about methods like op_Equality?

Reply:
Basically use nameof wherever you can.  If you can't use it in some circumstances, then don't :smile: